crates/compiler/test_syntax/tests/test_snapshots.rs

```diff
@@ -631,7 +734,7 @@ mod test_snapshots {
             Err(err) => Err(format!("{err:?}")),
         };
 
-        println!("{:?}", result);
+        // println!("{:?}", result);
```

Comment:
why was this removed?

Reply:
I actually forget why I had commented this out, but pretty sure it was just for cleanliness when doing debugging.